test/classes/ferguson/delete-free/owned-demo.chpl

```diff
@@ -56,28 +56,28 @@ proc examples() {
   var owned5 = new owned C(5)?;
   writeln("owned5 = ", owned5.borrow());
 
-  // The clear method can be used to delete the
+  // The release function can be used to delete the
   // managed object, if any.
-  owned5.clear(); // deletes C(5)
-  writeln("after owned5.clear()");
+  delete owned.release(owned5); // deletes C(5)
+  writeln("after delete owned.release(owned5)");
   writeln("owned5 = ", owned5.borrow());
 
-  // The retain method can be used to provide a new value for
+  // The adopt function can be used to provide a new value for
   // an owned to point to.
   // After this line:
   //  'owned1' will contain C(100); previous object C(4) will be destroyed
-  owned1.retain(new unmanaged C(100));
-  writeln("after owned1.retain C(100)");
+  owned1 = owned.adopt(new unmanaged C(100));
+  writeln("after owned1.adopt C(100)");
```

Comment:
Minor, but should this say `owned.adopt`?

Reply:
Hm, the code does not actually do "owned1.adopt" here. I prefer for it to say "owned1 = adopt ...."